### PR TITLE
969: create release notes for next releases

### PR DESCRIPTION
**Pull request by jorgesanchezperez**
- Added release notes template file
- Added release notes for latest release 1.0.1
Issue: https://github.com/SecureApiGateway/SecureApiGateway/issues/969

### CODE REVIEW

File: release-notes.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+[//]: # (<!-- always an empty line before table -->)
+
+## Secure API Gateway
+
+The ForgeRock Secure API Gateway for Open Banking is an open-source solution that helps financial services organizations deliver both test and production Open Banking and custom APIs from one framework, enforced by the ForgeRock Identity Platform.
+
+> These are the release notes for `Secure API Gateway` version `v1.0.1` named `Aerosmith`
+
+### Previous version
+- `v1.0.1`
```
Comment (dbadham-fr) [inline release-notes.md:14]: Should this be v1.0.0?

File: release-notes.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+[//]: # (<summary>Release Notes v1.0.1</summary>)
+
+[//]: # (<!-- always an empty line before table -->)
+
+## Secure API Gateway
+
+The ForgeRock Secure API Gateway for Open Banking is an open-source solution that helps financial services organizations deliver both test and production Open Banking and custom APIs from one framework, enforced by the ForgeRock Identity Platform.
+
+> These are the release notes for `Secure API Gateway` version `v1.0.1` named `Aerosmith`
```
Comment (dbadham-fr) [inline release-notes.md:11]: I think this would look better as a regular paragraph rather than a quote block

File: release-notes.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+### Release Notes v1.0.1
+
+### Upgrade Steps
+[n/a]
+
+### Breaking Changes
+[n/a]
+
+### New Features
+#### Addressed Issues
```
Comment (dbadham-fr) [inline release-notes.md:33]: Remove the Addressed Issues sub heading, I think the New Features heading is sufficient

File: release-notes.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+One that uses OB Certs and one that uses IG issues certs, and the rest of the collection would work which ever type of certs you were using.
+
+[947: Display debtor account preselected when is provided and update the decision object](https://github.com/secureapigateway/secureapigateway/issues/947)
+
+In the payment initiation, the debtor account is optional as the PISP may not know the account
+identification details for the PSU.
+
+When the PISP knows the account identification, the debtor account is provided in the consent.
+
+Now the Resource Consent Service user App, displays the consent details with a preselected
```
Comment (dbadham-fr) [inline release-notes.md:58]: Resource Consent Service user App => Remote Consent Service UI

**Review by dbadham-fr — COMMENTED** [review]
Looks nice. 

I've made a few suggestions, mostly about formatting/presentation.

File: release-notes.md (hunk 1 above)
Comment (jorgesanchezperez) [reply release-notes.md]: Ups yes! fixed

File: release-notes.md (hunk 2 above)
Comment (jorgesanchezperez) [reply release-notes.md]: Done!

File: release-notes.md (hunk 3 above)
Comment (jorgesanchezperez) [reply release-notes.md]: done

File: release-notes.md (hunk 4 above)
Comment (jorgesanchezperez) [reply release-notes.md]: Done

**Pull request closed without merging**